serverlessworkflow/modules/ROOT/pages/cloud/operator/using-persistence.adoc

```diff
@@ -29,82 +36,261 @@ spec:
   persistence:
     postgresql:
       secretRef:
-        name: postgres-secrets
-        userKey: POSTGRES_USER
-        passwordKey: POSTGRES_PASSWORD
+        name: postgres-secrets <1>
+        userKey: POSTGRESQL_USER <2>
+        passwordKey: POSTGRESQL_PASSWORD <3>
       serviceRef:
-        name: postgres
-        port: 5432
-        databaseName: sonataflow
-        databaseSchema: shared
-  build:
-    config:
-      strategyOptions:
-        KanikoBuildCacheEnabled: "true"
----
-
-The values of `POSTGRES_USER` and `POSTGRES_PASSWORD` are the keys in the [Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/) that contains the credentials to connect to the postgreSQL instance. 
-The SonataFlow Workflow CR 
-is defined with its `Persistence` field defined as empty.
+        name: postgres <4>
+        namespace: sonataflow-infra <5>
+        databaseName: sonataflow <6>
+        port: 1234 <7>
+----
+
+<1> Name of the link:{k8n_secrets_url}[Kubernetes Secret] containing the username and password to connect with the database.
+<2> Name of the link:{k8n_secrets_url}[Kubernetes Secret] `key` containing the username to connect with the database.
+<3> Name of the link:{k8n_secrets_url}[Kubernetes Secret] `key` containing the password to connect with the database.
+<4> Name of the Kubernetes Service to connect with the PostgreSQL database server.
+<5> (Optional) Kubernetes namespace containing the PostgreSQL Service. Defaults to the SonataFlowPlatform's local namespace.
+<6> Name of the PostgreSQL database to store the workflow's data.
+<7> (Optional) Port number to connect with the PostgreSQL Service. Defaults to 5432.
+
+This configuration signals the operator that every workflow deployed in current SonataFlowPlatform's namespace must be properly configured to connect with that PostgreSQL database server.
+And the operator will add the relevant JDBC connection parameters in the form of environment variables to the workflow container.
+
+Additionally, for SonataFlow CR deployments that use the `preview` profile, it will configure the {product_name} build system to include specific Quarkus extensions required for persistence.
```

Comment:
```suggestion
Additionally, for `SonataFlow` CR deployments that use the `preview` profile, it will configure the {product_name} build system to include specific Quarkus extensions required for persistence.
```